prime-router/src/main/kotlin/Translator.kt

```diff
@@ -106,6 +106,11 @@ class Translator(private val metadata: Metadata, private val settings: SettingsP
         // ok to just return null, we don't need any info about what was eliminated.
         if (jurisFilteredReport.isEmpty()) return null
 
+        // keep track of how many items passed the juris filter prior to quality filtering.
+        // For informational/reporting purposes only.
+        // Normally this value is passed from parent to child Report, like mitochondrial DNA.  This overrides that.
+        jurisFilteredReport.itemCountBeforeQualFilter = jurisFilteredReport.itemCount
+
```

Comment:
This is the key place where we snag the value -- after juris filtering and before qual filtering.

Pretty much all the other uses are just passing the value hither and yon.